hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientManager.java

```diff
@@ -232,10 +225,10 @@ public XceiverClientSpi call() throws Exception {
             switch (type) {
             case RATIS:
               client = XceiverClientRatis.newXceiverClientRatis(pipeline, conf,
-                  caCert);
+                  caCerts);
               break;
             case STAND_ALONE:
-              client = new XceiverClientGrpc(pipeline, conf, caCert);
+              client = new XceiverClientGrpc(pipeline, conf, caCerts);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
Analysis of this package determined that this line of code contains a resource that might not have closed properly. A resource leak can slow down or crash your system.

**Fix**
Consider closing the following resource: *client*. The resource is referenced at line 237. The resource is returned at line 238. There are other execution paths that don&#39;t close the resource or return it, for example, when XceiverClientSpi.connect throws an exception. To prevent this resource leak, close *client* along these other paths before you exit this method.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).